doc/modules/changes/20220630_haoyuan

```diff
@@ -0,0 +1,9 @@
+New: The values of the properties of material phases can be entered
+by the differences between adjacent phases. If this is the case,
```

Comment:
Please leave a note here why this may be needed or a good idea. From the conversation in this PR I gather you have discussed that at some point with @jdannberg or @anne-glerum, but I (and other ASPECT users) were not part of that conversation, and the code itself does not seem to mention it. Without this explanation this looks like a whole lot of complication.